src/trainers/mitie_sklearn_trainer.py

```diff
@@ -28,45 +27,46 @@ def start_and_end(self, text_tokens, entity_tokens):
         return start, end
 
     def train_entity_extractor(self, entity_examples):
-        trainer = ner_trainer(self.fe_file)
-        for example in entity_examples:
-            tokens = tokenize(example["text"])
-            sample = ner_training_instance(tokens)
-            for ent in example["entities"]:
-                _slice = example["text"][ent["start"]:ent["end"] + 1]
-                val_tokens = tokenize(_slice)
-                start, end = self.start_and_end(tokens, val_tokens)
-                sample.add_entity(xrange(start, end), ent["entity"])
-            trainer.add(sample)
+        self.entity_extractor = mitie_trainer_utils.train_entity_extractor(entity_examples,
+                                                                           self.fe_file,
+                                                                           self.max_num_threads,)
 
-        ner = trainer.train()
-        return ner
+    def train_intent_classifier(self, intent_examples, test_split_size=0.1):
+        self.intent_classifier = SklearnIntentClassifier(max_num_threads=self.max_num_threads)
```

Comment:
Is there a reason not to use `sklearn_trainer_utils.train_intent_classifier` here?